backing-services/docker-compose.yml

```diff
@@ -38,4 +38,13 @@ services:
       - auth
       - consul
     cap_add:
-      - IPC_LOCK
\ No newline at end of file
+      - IPC_LOCK
+
+  isso:
+    image: docker.netsoc.co/public/isso:dev-env
+    container_name: isso
+    ports:
+      - 8181:8080
+    depends_on:
+      - db
+    restart: always
```

Comment:
remove this pls. the container will be started on system startup with this even if im not gonna be working on netsoc stuff